src/state/utils/store_io.ts

```diff
@@ -123,6 +123,7 @@ export const serializedDataToUrl = (data: SerializedStoreData): string => {
  * called after the collection and dataset are loaded and set in the store.
  */
 export const loadViewerStateFromParams = async (store: Store<ViewerStore>, params: URLSearchParams): Promise<void> => {
+  // TODO: Should each of these be wrapped in a try/catch block in case of bad inputs?
```

Comment:
Update: I'm going to address this in a follow-up PR! I'll add an error banner to handle this, with a convenient link for submitting a new issue :')